password_generator.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+import tkinter as tk
+import random
+import string
+
+#create class to use for the generate button
+class Password:
+	def __init__(self, length: int):
+		self.length = length
+	def generate(self):
+		return ''.join(random.choices(string.ascii_letters + string.digits + string.punctuation, k=self.length))
+	@property
+	def verification(self):
+		while True:
+			passw = self.generate()
+			c = any( a == True for a in [i.islower() for i in passw])
+			d = any( a == True for a in [i.isupper() for i in passw])
+			e = any( a == True for a in [i.isdigit() for i in passw])
+			f = set(passw).difference(string.ascii_letters,string.digits)
+			if c == False: #check if generated password contains lowercase, if not continue
```

Comment:
because `c` is a boolean, you can do:
```suggestion
			#check if generated password contains lowercase, if not continue
			if not c:
```
Once you combine that with better variable names, you start to get code that can be read like plain english